### ISSUE TITLE
CORS not working

### ISSUE THREAD

**Issue by mausamadh** [sanic-org/sanic-guide#70]
https://sanicframework.org/en/guide/how-to/cors.html#options.py

        for route in routes.values():
                for uri, methods in route.methods.items(): 
                    if "OPTIONS" not in methods:
                        needs_options[uri].extend(methods)

getting error in the second loop, it says frozenset has no attribute items.

**Comment by sjsadowski** [comment]
change
```
for uri, methods in route.methods.items():
```
to
```
for uri, methods in route.methods:
```

**Comment by ahopkins** [comment]
I don't think that is a tuple that can be deconstructed. You'd have to use `route.uri`. I'll post a fix tomorrow.
